src/engine/CallFixedSize.h

```diff
@@ -62,10 +64,10 @@ auto callLambdaForIntArray(std::array<Int, NumValues> array, auto&& lambda,
 
   // Call the `lambda` when the correct compile-time `Int`s are given as a
   // `std::integer_sequence`.
-  auto applyOnIntegerSequence =
-      [&]<Int... Is>(ad_utility::ValueSequence<Int, Is...>) {
-        return lambda.template operator()<Is...>(AD_FWD(args)...);
-      };
+  auto applyOnIntegerSequence = CPP_template_lambda(&)(Int... Is)(
```

Comment:
In this case, the easiest way is to probably make ` applyOnIntegerSequence` a free function, that explicitly takes the `lambda, the ` ValueSequence` and the ` args` .

Reply:
Done